staging/src/k8s.io/apiserver/pkg/util/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	subsystem  = "apiserver"
+	statuscode = "code"
+)
+
+var registerMetricsOnce sync.Once
+
+var (
+	// streamTranslatorRequestsTotal counts the number of requests that were handled by
+	// the StreamTranslatorProxy.
+	streamTranslatorRequestsTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      subsystem,
+			Name:           "stream_translator_request_total",
```

Comment:
following naming guide in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/instrumentation.md#instrumenting-kubernetes and https://prometheus.io/docs/practices/naming/, looks like this should be `stream_translator_requests_total` (plural)

Reply:
Fixed.